terminus/tests/road_test.py

```diff
@@ -66,3 +68,13 @@ def test_length_empty(self):
     def test_length_empty_road(self):
         road = Street.from_points([])
         self.assertEqual(road.length(), 0.0)
+
+    def test_node_bounding_boxes(self):
```

Comment:
It would be nice to think the different cases that need testing. I would suggest starting with a single lane with offset 0, then a single lane with offset not 0 and finally one with multiple lanes (having positive and negative offsets).